core/tests/test_views.py

```diff
@@ -253,12 +253,27 @@ def test_privacy_cookies_cms(
     assert response.template_name == [expected_template]
 
 
+@pytest.mark.parametrize(
+    'activated_language,component_languages,dir',
+    (
+        (
+            'ar', [['ar', 'العربيّة'], ('en-gb', 'English')], 'rtl'
+        ),
+        (
+            'en-gb', [['ar', 'العربيّة'], ('en-gb', 'English')], 'ltr'
+        ),
+        (
+            'zh-hans', [['ar', 'العربيّة'], ('en-gb', 'English')], 'ltr'
+        ),
+    )
+)
 @patch('core.views.InternationalLandingPageView.cms_component',
        new_callable=PropertyMock)
 @patch('core.views.InternationalLandingPageView.page',
        new_callable=PropertyMock)
 def test_international_landing_page_news_section_on(
-    mock_get_page, mock_get_component, client, settings
+    mock_get_page, mock_get_component, activated_language,
+    component_languages, dir, client, settings
```

Comment:
can you please use a different variable name for `dir` - `dir` is actually a [built-in function](https://www.programiz.com/python-programming/methods/built-in/dir)